packages/rocketchat-lib/client/lib/openRoom.js

```diff
@@ -66,7 +66,9 @@ function openRoom(type, name) {
 
 			Session.set('openedRoom', room._id);
 			RocketChat.openedRoom = room._id;
-
+			const room_type = RocketChat.roomTypes.roomTypes[type].route.name;
+			const room_path = `/${ room_type }/${ name }`;
```

Comment:
Instead of combining these values into a string, I believe it would be better to instead just pass in the room id.